src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/hearings/hearingdetails/HearingDetailsService.java

```diff
@@ -35,12 +35,11 @@ public void initialiseHearingDetails(CaseData caseData) {
     }
 
     public void handleListingSelected(CaseData caseData) {
-
         HearingType selectedHearing = getSelectedHearing(caseData);
         HearingDetailTypeItem hearingDetailTypeItem;
         HearingDetailType hearingDetailType;
         List<HearingDetailTypeItem> hearingDetailTypeItemList = new ArrayList<>();
-        if (selectedHearing != null && !CollectionUtils.isEmpty(selectedHearing.getHearingDateCollection())) {
```

Comment:
Not sure if selectedHearing null check will still make sense?

Reply:
added